lib/classifier-reborn/backends/bayes_redis_backend.rb

```diff
@@ -1,4 +1,8 @@
-require 'redis'
+begin
+  require 'redis'
+rescue LoadError
+  puts 'The redis gem is required to use the redis backend.'
```

Comment:
Should we use `$stderr` instead? Unless, this repository as a whole only uses STDOUT for all sorts of outputs.

```ruby
$stderr.puts "The redis gem is required to use the Redis backend. To install redis gem run:\n\ngem install redis\n"
```